asyncio port scanner.py

```diff
@@ -0,0 +1,31 @@
+import time
+import asyncio
+
+
+async def connect_socket(ip: str, port: int):
+    """
+    The function try to connect a given socket to a given ip and port. If not
+    succeeded it raise an exception and print a massage.
```

Comment:
Simplify the docstring of the method. It can be shortened to 1 line title.

```suggestion
    Connect to given IP and port
```

Note the grammar. I am using infinitive verbs to indicate what the function **should do** when called. Also I am not putting a dot at the end because it is a title summary. It is not a sentance per say; but a header; כותרת.